[All hosts](security docs refresh) (#3010)
OfficeDev/office-js-docs-pr: docs/develop/auth-external-add-ins.md

---
title: Authorization with non-Microsoft identity providers
description: 'Get authorization to non-Microsoft data sources using OAuth 2.0, the Authorization Code and Implicit flows.'
ms.date: 10/22/2021
ms.localizationpriority: medium
---

# Authorization with non-Microsoft identity providers

> There are many popular online services outside of the Microsoft identity platform that are identity providers that developers can use. They give users access to their accounts in other applications, including your Office Add-in.

Reviewer: ```suggestion
There are many popular identity providing services, in addition to the Microsoft identity platform, that you can use in your add-in. They give users, and applications such as your Office add-in, access to the users' accounts in other applications.
```

---
title: Authorization with Google, Facebook, and other identity providers

> description: 'Getting authorization to non-Microsoft data sources such as Google, Facebook, LinkedIn, SalesForce, and GitHub using OAuth 2.0, the Authorization Code and Implicit flows.'

Author: fixed
